### PR TITLE
LPS-157627 Test Automation Creation - master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-157627

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 1]
```diff
@@ -2297,6 +2297,118 @@ definition {
...
+					key_text = "Success Text",
+					locator1 = "TextInput#ANY",
+					value1 = "${successText}");
+
+				Click(
+					key_sidebarButton = "Browser",
+					locator1 = "PageEditor#FRAGMENT_SIDEBAR_TITLE");
+
+				PageEditor.waitForAutoSave();
+			}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:2352]: Move into if ("${successMessage}" == "Embedded") {

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 2]
```diff
@@ -2297,6 +2297,118 @@ definition {
...
+					key_text = "URL",
+					locator1 = "TextInput#ANY",
+					value1 = "${url}");
+
+				Click(
+					key_sidebarButton = "Browser",
+					locator1 = "PageEditor#FRAGMENT_SIDEBAR_TITLE");
+
+				PageEditor.waitForAutoSave();
+			}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:2405]: Move into 			else if ("${successMessage}" == "URL") {

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 3]
```diff
@@ -2297,6 +2297,118 @@ definition {
...
+
+			if ("${previewSuccessState}" == "true") {
+				Check.checkNotVisible(
+					checkboxName = "Preview Success State",
+					locator1 = "Checkbox#ANY_CHECKBOX");
+			}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:2411]: Move into if ("${successMessage}" == "Embedded") {

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 4]
```diff
@@ -2297,6 +2297,118 @@ definition {
...
+					locator1 = "Treeview#NODE_ITEM");
+
+				Click(
+					key_nodeName = "${pageName}",
+					locator1 = "Treeview#NODE_UNSELECTED");
+
+				PageEditor.viewItemSelectorText(
+					fieldLabel = "Page",
+					text = "${pageName}");
+			}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:2392]: Move into if ("${successMessage}" == "Page") { and rename to if (isSet(pageName)) {

Use
```
			if (IsTextEqual(key_fieldLabel = "Page", locator1 = "PageEditor#ITEM_SELECTOR_CONTENT_INPUT", value1 = "")) {
				Click(
					key_fieldLabel = "Page",
					key_text = "plus",
					locator1 = "Icon#ANY_ITEM_SELECTOR");
			}
			else {
				Click(
					key_fieldLabel = "Page",
					key_text = "change",
					locator1 = "Icon#ANY_ITEM_SELECTOR");
			}

```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FormContainer.testcase [hunk 5]
```diff
@@ -222,9 +243,512 @@ definition {
...
+		task ("When the page designer checks preview success state toggle") {
+			PageEditor.editFormContainer(
+				panel = "General",
+				previewSuccessState = "true");
+		}
+
+		task ("Then the page designer can see sucess text under Form Container") {
+			AssertTextEquals(
+				locator1 = "Fragment#FORM_CONTAINER_SUCCESS_MESSAGE",
+				value1 = "Thank you. Your information was successfully received.");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/formcomponents/FormContainer.testcase:303]: Use 
```
			AssertTextPresent(value1 = "Thank you. Your information was successfully received.");

```

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/fragments/Fragment.path [hunk 6]
```diff
@@ -140,6 +140,11 @@
 
 <!--FORM COMPONENTS-->
 
+<tr>
+	<td>FORM_CONTAINER_SUCCESS_MESSAGE</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/fragments/Fragment.path:144]: No need

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao! PR updated, would you have a look at? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 7]
```diff
@@ -2297,6 +2297,114 @@ definition {
...
+						key_nodeName = "${pageName}",
+						locator1 = "Treeview#NODE_ITEM");
+
+					Click(
+						key_nodeName = "${pageName}",
+						locator1 = "Treeview#NODE_UNSELECTED");
+
+					PageEditor.viewItemSelectorText(
+						fieldLabel = "Page",
+						text = "${pageName}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:2379]: Remove this validation step from operation macro. Add `SelectFrame(value1 = "relative=top");`

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating?

**Comment by Tim-Cao** [comment]
See changes @ozysouza

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**